docs/conf.py

```diff
@@ -189,11 +197,59 @@
 from gammapy.utils.docs import gammapy_sphinx_ext_activate
 gammapy_sphinx_ext_activate()
 
-# copy notebooks
-if os.environ['GAMMAPY_EXTRA']:
+# ---
+# notebooks
+# make relative links
+DOWNLOAD_CELL = """
+<div class='admonition note'>
+[Download notebook](../_static/notebooks/nbfilename.ipynb)
+
+This is a *fixed-text* formatted version of a Jupyter notebook. """
+if on_rtd:
+    DOWNLOAD_CELL += """
+    You may download the whole HTML documentation for this version of gammapy
+    using the link at the bottom of this page and execute the notebooks
+    in your local desktop inside the `_static/notebooks/` folder.
+    """
+DOWNLOAD_CELL += """</div>"""
+
+url_docs = setup_cfg.get('url_docs')
+
+def modif_links(folder):
+    for filename in os.listdir(folder):
+        filepath = os.path.join(folder, filename)
+        if os.path.isfile(filepath) and filepath[-6:] == '.ipynb':
+            if folder=='notebooks':
+                strcell = DOWNLOAD_CELL.replace('nbfilename.ipynb', filename)
```

Comment:
I would suggest to use `str.format` instead of `str.replace` to fill things in.
I.e. something like this:
template = """
Download file: {nb_filename}.
"""
and then
```
ctx = dict(nb_filename=nb_filename) # gather things in a "context" dict if you have many
txt = template.format(**ctx) # pass context to template.format
```